overlord/snapstate/backend.go

```diff
@@ -50,6 +50,7 @@ type StoreService interface {
 	DownloadStream(context.Context, string, *snap.DownloadInfo, int64, *auth.UserState) (r io.ReadCloser, status int, err error)
 
 	Assertion(assertType *asserts.AssertionType, primaryKey []string, user *auth.UserState) (asserts.Assertion, error)
+	DownloadAssertions([]string, *asserts.Batch, *auth.UserState) error
```

Comment:
I think that code completion/doc helpers display the function signatures from the interface (at least that's what happens in my Emacs setup). It's nice to provide parameter names in the interface so that it pop up in the doc hint.

```suggestion
	DownloadAssertions(streamURLs []string, batch *asserts.Batch, user *auth.UserState) error
```

Reply:
there's more atm that don't have arguments though, it's probably material for a follow up